apps/dashboard_web/lib/dashboard_web/live/components/check_in_count.ex

```diff
@@ -24,8 +24,9 @@ defmodule DashboardWeb.Components.CheckInCount do
       |> Map.put(:title, "Check-ins")
       |> Map.put(:product, :checkins)
       |> Map.put(:grid_width, 1)
-      |> Map.put(:data_key, :check_in_count)
       |> Map.put(:data_number, total_count)
+      |> Map.put(:timeframe, "This week")
+      |> Map.put(:comparison_number, 800)
```

Comment:
Is this just a default comparison number? If so it should probably be `0`.

Reply:
It is just me transferring your `arbitrary_comparison_number` from here: https://github.com/geolessel/pco-dashboard/pull/28/files#diff-23618e65c70c9ee74c567191884882f3L17